### PR TITLE
Shortened dev mode urls

### PR DESCRIPTION
**Pull request by brandenrodgers**
## Description and Context

More context on the issue here: https://github.com/HubSpot/hubspot-cli/pull/835#issuecomment-1533240834

Here's one example of a way that we could handle long URLs that don't play nicely with the spinnies lib. This uses our running server to redirect urls.

### Before
image

### After
image

### CODE REVIEW

File: packages/cli/lib/LocalDevManager.js [hunk 1]
```diff
@@ -159,14 +160,10 @@ class LocalDevManager {
...
     });
-    const projectDetailUrl = getProjectDetailUrl(
-      this.projectConfig.name,
-      this.targetAccountId
-    );
     this.spinnies.add('viewInHubSpotLink', {
       text: uiLink(
         i18n(`${i18nKey}.header.viewInHubSpotLink`),
-        projectDetailUrl,
+        `${this.devServerPath}/hs/project`,
```
Comment (kemmerle) [inline packages/cli/lib/LocalDevManager.js:166]: What happens if the `this.devServerPath` remains `null`? Is that possible?

File: packages/cli/lib/LocalDevManager.js [hunk 2]
```diff
@@ -510,23 +509,24 @@ class LocalDevManager {
   }
 
   async startServers() {
-    // TODO spin up local dev servers
-    return true;
+    this.devServerPath = await DevServerManager.start({
```
Comment (kemmerle) [inline packages/cli/lib/LocalDevManager.js:512]: Should we be declaring this above line 166, where we reference `this.devServerPath`?

**Review by kemmerle — APPROVED** [review]
I had two small questions which I believe should require only minor work, if any. Otherwise, LGTM :shipit:

File: packages/cli/lib/LocalDevManager.js (hunk 2 above)
Comment (brandenrodgers) [reply packages/cli/lib/LocalDevManager.js:512]: This `startServers` function actually [does get called before the code on line 166 executes](https://github.com/HubSpot/hubspot-cli/pull/850/files#diff-0391b654b5bab2f8d2b05906b406db63c1c591cc6b937dfba92d6a7d475cf878R93). So we should be all set there

File: packages/cli/lib/LocalDevManager.js (hunk 1 above)
Comment (brandenrodgers) [reply packages/cli/lib/LocalDevManager.js]: done in [fe1c962](https://github.com/HubSpot/hubspot-cli/pull/850/commits/fe1c9622912af7e681d4f05cb2e3484b5849afd9)

**Pull request merged**